spec/unit/blockscore/person_spec.rb

```diff
@@ -1,10 +1,12 @@
 module BlockScore
   RSpec.describe Person do
```

Comment:
As described in another PR, I would probably not monkey patch `RSpec.describe` inside the BlockScore namespace. By including it inside this namespace you're inadvertently modifying the object under test.

Also if you use the suggested `RSpec.describe BlockScore::Person`, then you can replace the places within the spec that refer to `BlockScore::Person` directly with `describe_class`.